Source/DataSources/Angle.js

```diff
@@ -0,0 +1,97 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/Event'
+    ], function(
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        Event) {
+    "use strict";
+
+    /**
+     * A property type whose value is always interpolated by taking the the shortest distances between 2 angles.
+
+     *
+     * @alias ConstantProperty
+     * @constructor
```

Comment:
This should be `@namspace` instead of `@constructor` because it is not a class meant to be instantiated.